livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -64,37 +106,19 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
         return new InterpreterResult(InterpreterResult.Code.SUCCESS, "");
       }
 
-      // create sqlContext implicitly if it is not available, as in livy 0.2 sqlContext
-      // is not available.
-      synchronized (this) {
-        if (!sqlContextCreated) {
-          InterpreterResult result = sparkInterpreter.interpret("sqlContext", context);
-          if (result.code() == InterpreterResult.Code.ERROR) {
-            result = sparkInterpreter.interpret(
-                "val sqlContext = new org.apache.spark.sql.SQLContext(sc)\n"
-                    + "import sqlContext.implicits._", context);
-            if (result.code() == InterpreterResult.Code.ERROR) {
-              return new InterpreterResult(InterpreterResult.Code.ERROR,
-                  "Fail to create sqlContext," + result.message());
-            }
-          }
-          sqlContextCreated = true;
-        }
+      // use triple quote so that we don't need to do string escape.
+      String sqlQuery = null;
+      if (isSpark2) {
+        sqlQuery = "spark.sql(\"\"\"" + line + "\"\"\").show(" + maxResult + ")";
```

Comment:
“”“ is OK for line separator. And I remember I try `StringEscapeUtils` before but hit some weird issue.